src/protections/NsfwProtection.ts

```diff
@@ -45,71 +45,62 @@ export class NsfwProtection extends Protection {
     }
 
     public async handleEvent(mjolnir: Mjolnir, roomId: string, event: any): Promise<any> {
-        if (event["type"] === "m.room.message") {
-            let content = JSON.stringify(event["content"]);
-            if (!content.toLowerCase().includes("mxc")) {
-                return;
-            }
-            // try and grab a human-readable alias for more helpful management room output
-            const maybeAlias = await mjolnir.client.getPublishedAlias(roomId);
-            const room = maybeAlias ? maybeAlias : roomId;
+        if (event.type !== "m.room.message" && event.type !== "m.sticker") {
+            return;
+        }
```

Comment:
This will make the protection exclusively scan stickers which i don't think this was intended? meow?

Reply:
ok mb i can't read

Reply:
i read this through twice and it looks okay :)